UnitTesting/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj

```diff
@@ -49,26 +48,39 @@
       <HintPath>..\packages\Moq.4.14.1\lib\net45\Moq.dll</HintPath>
     </Reference>
     <Reference Include="PX.Common">
-      <HintPath>..\..\..\..\Bin\PX.Common.dll</HintPath>
+      <HintPath>..\..\..\..\..\PhoneRepairShopT280\Bin\PX.Common.dll</HintPath>
     </Reference>
     <Reference Include="PX.Common.Std">
-      <HintPath>..\..\..\..\Bin\PX.Common.Std.dll</HintPath>
+      <HintPath>..\..\..\..\..\PhoneRepairShopT280\Bin\PX.Common.Std.dll</HintPath>
     </Reference>
     <Reference Include="PX.Data">
-      <HintPath>..\..\..\..\Bin\PX.Data.dll</HintPath>
+      <HintPath>..\..\..\..\..\PhoneRepairShopT280\Bin\PX.Data.dll</HintPath>
     </Reference>
     <Reference Include="PX.Data.BQL.Fluent">
-      <HintPath>..\..\..\..\Bin\PX.Data.BQL.Fluent.dll</HintPath>
+      <HintPath>..\..\..\..\..\PhoneRepairShopT280\Bin\PX.Data.BQL.Fluent.dll</HintPath>
     </Reference>
     <Reference Include="PX.Objects">
-      <HintPath>..\..\..\..\Bin\PX.Objects.dll</HintPath>
+      <HintPath>..\..\..\..\..\PhoneRepairShopT280\Bin\PX.Objects.dll</HintPath>
     </Reference>
     <Reference Include="PX.Tests.Unit">
-      <HintPath>..\..\..\..\Bin\PX.Tests.Unit.dll</HintPath>
+      <HintPath>..\..\..\..\..\PhoneRepairShopT280\App_Data\Projects\PhoneRepairShop\PX.Tests.Unit.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Buffers, Version=4.0.3.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
```

Comment:
I am not sure. System.Buffers does not seem to be a direct Autofac dependency. I only installed the packages that were needed in the course. It automatically downloads other dependencies. The old project for T280 on github had fewer dependencies but I did not want to add only those because some dependencies may have changed since in those packages. I added all the packages in my packages folder to ensure the project doesn't break due to some missing dependency later.